File: Infoteca_cu/connection.php
<?php

    $servername = "localhost";
    $username = "mariomanza";
    $password = "sexymexy45!";
    $dbname = "northwind";

try {
    $conn = new PDO("mysql:host=$servername;dbname=$dbname", $username, $password);
    $conn->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
    #prepared statement for query by date range
    $stbd = $conn->prepare("select concat(FirstName,' ', LastName) as Full_name, City, year(HireDate) as year from employees where HireDate between ? and ?;");
    $stbd->bindParam(1, $date_s);
    $stbd->bindParam(2, $date_f);
    $stbd->setFetchMode(PDO::FETCH_ASSOC);
    $results = $stbd->fetchAll();
} catch(PDOException $e) {
        echo "Connection failed: " . $e->getMessage();
    }

    $results;
    $id = 1;
    $date_s = date("Y-m-d");
    $date_f = date("Y-m-d");

    if($_SERVER["REQUEST_METHOD"] == "POST") {
        $date_s = test_input($_POST["date_s"]);
        $date_f = test_input($_POST["date_f"]);
        $stbd->execute();
        $results = $stbd->fetchAll();
    }

function by_id(){
    if($_SERVER["REQUEST_METHOD"] == "POST") {
        $id = test_input($_POST["id"]);
        $stbi->execute($id);
        $results = $stbi->fetchAll();
    }
}

function test_input($data) {
    $data = trim($data);
    $data = stripslashes($data);
    $data = htmlspecialchars($data);
    return $data;
}

?>

File: Infoteca_cu/connection.py
import django
import mysql.connector

db = mysql.connector.connect(host="localhost", user="mariomanza",
                             passwd="sexymexy45!",
                             database="northwind")

date_sql = "select concat(FirstName,' ', LastName) as Full_name, City, year(HireDate) as year from employees where HireDate between %s and %s;"
id_sql = "select concat(firstname,' ', lastname) as Full_name, City, year(Hiredate) as year from employees where id = %s"
cursor = db.cursor()

ch = int(input('Enter option: '))
if ch == 1:
    cursor.execute("show databases")
elif ch == 2:
    cursor.execute("show tables")
elif ch == 3:
    print(cursor.rowcount)
    print(cursor.lastrowid)
elif ch == 4:
    cursor.execute("select firstname, lastname from employees")
elif ch == 5:
    date_s = input('Enter start date:')
    date_f = input('Enter final date:')
    date = (date_s,date_f)
    cursor.execute(date_sql, date)
else:
    print('select an option')

result = cursor.fetchall()
for x in result:
    print(x)
